request_store.gemspec

```diff
@@ -14,8 +14,6 @@ Gem::Specification.new do |gem|
   gem.licenses      = ["MIT"]
 
   gem.files         = `git ls-files`.split($/)
-  gem.executables   = gem.files.grep(%r{^bin/}).map{ |f| File.basename(f) }
-  gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
```

Comment:
I would prefer to keep these, though I'm happy to get rid of the executables.

Reply:
Are _you_ aware of any current use of them?

Reply:
So, back in the Perl days, when you'd install a package, it would also run your tests, to make sure it actually works on your platform. Ruby culture didn't inherit this particular thing, so it's not actually a big deal. I personally really like to try and keep published artifacts as close to what's in the repo as possible.

Similar to your other PR, I should probably just get over it :)

Reply:
Thank you for offering the personal connection to the directive. Perl gave us so much.

Reply:
@steveklabnik OK! Now, I edited the commit to leave the test_files directive in place.